cmd/registry/cmd/get/get.go

```diff
@@ -75,9 +77,19 @@ func Command() *cobra.Command {
 				Pattern:        pattern,
 				Filter:         filter,
 			}); err != nil {
+				if status.Code(err) == codes.NotFound {
+					fmt.Fprintln(cmd.ErrOrStderr(), "Not Found")
```

Comment:
This is an interesting detail as well... normally this cmd emits yaml. If this cmd is being used for scripting, should the output be nothing? Or just the yaml header? It seems that non-yaml might not be the right choice?

Reply:
Never mind, this is printed to stderr.

Reply:
Right, I thought we should keep stdout clear here.